src/jest-extensions.js

```diff
@@ -9,10 +17,30 @@ function getDisplayName(subject) {
   }
 }
 
+function checkHtmlElement(htmlElement) {
+  if (!(htmlElement instanceof HTMLElement)) {
+    throw new Error(
+      `The given subject is a ${getDisplayName(
+        htmlElement,
+      )}, not an HTMLElement`,
+    )
+  }
+}
+
 const assertMessage = (assertionName, message, received, expected) =>
   `${matcherHint(`${assertionName}`, 'received', '')} \n${message}: ` +
   `${printExpected(expected)} \nReceived: ${printReceived(received)}`
 
+function printAttribute(name, value) {
+  return value === undefined ? name : `${name}=${stringify(value)}`
+}
+
+function getAttributeComment(name, value) {
+  return value === undefined
+    ? `element.hasAttribute(${stringify(name)})`
+    : `element.getAttribute(${stringify(name)}) === ${stringify(value)}`
+}
+
 const extensions = {
   toBeInTheDOM(received) {
     getDisplayName(received)
```

Comment:
I just noticed that this actually isn't doing anything. I'm pretty sure we should assign it a value called `displayName` and use that somehow or remove it entirely... Probably good for another PR though.

Reply:
Yes, I noticed that too, but I preferred no to touch it here as it's unrelated.

I do have a few code cleanup proposals for this file, that I was meaning to provide in a separate PR. For instance, the pattern of returning something different on the matchers depending on wether they `pass`ed or not seems redundant to me.

Reply:
Yes, I'd love it if you could clean this file up a bit in another PR! Let's just see what we need to do to increase coverage back to 100%.

Open `coverage/lcov-report/index.html` in your browser to see what's left :+1: